source/MaterialXRenderGlsl/GLCocoaWrappers.m

```diff
@@ -0,0 +1,221 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXRender/HardwarePlatform.h>
+
+#if defined (OSMac_)
+
+#import <Cocoa/Cocoa.h>
+#import <AppKit/NSApplication.h>
+#import "GLCocoaWrappers.h"
+
+void* NSOpenGLChoosePixelFormatWrapper(bool allRenders, int bufferType, int colorSize, int depthFormat,
+								int stencilFormat, int auxBuffers, int accumSize, bool minimumPolicy,
+								bool accelerated, bool mp_safe,  bool stereo, bool supportMultiSample)
+{
+	// Create local autorelease pool for any objects that need to be autoreleased.
+	NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
+
+	NSOpenGLPixelFormatAttribute list[50];
+	int i = 0;
+	if (allRenders)
```

Comment:
Mixed tabs and spaces in this file, and these should all be converted to spaces.